packages/block-library/src/navigation/edit/index.js

```diff
@@ -257,6 +269,17 @@ function Navigation( {
 		hasResolvedNavigationMenus &&
 		! hasUncontrolledInnerBlocks;
 
+	if ( isPlaceholder && ! ref ) {
```

Comment:
```suggestion
	if ( isFallback && ! ref ) {
```

Do you think we ought now to rename the term "placeholder" to avoid it being confused with `<Placeholder>` components, especially as we will provide one of these components for backwards compat.

Reply:
I did think about this as well but the page list block is a placeholder and we also have the custom placeholder that depends on `isPlaceholder`.